zkevm-circuits/src/evm_circuit.rs

```diff
@@ -457,7 +478,7 @@ mod evm_circuit_stats {
     fn evm_circuit_unusable_rows() {
         assert_eq!(
             EvmCircuit::<Fr>::unusable_rows(),
-            unusable_rows::<Fr, EvmCircuit::<Fr>>(()),
+            unusable_rows::<Fr, EvmCircuit::<Fr>>(FeatureConfig::default()),
```

Comment:
I wonder if different feature configs could lead to different unusable rows :thinking: